src/components/timeline/index.css

```diff
@@ -12,7 +12,7 @@
   background: linear-gradient(
     to left,
     white,
-    #eee calc(var(--vertical-scrollbar-reserved-width) - 1px),
+    #eee var(--vertical-scrollbar-reserved-width),
     var(--grey-30) 0,
     var(--grey-30) var(--vertical-scrollbar-reserved-width),
```

Comment:
This change makes the `--grey-30` border disappear. Was this intentional?

Reply:
Oh, that wasn't intentional. Thanks for cathing it! Reverted it.